docker-compose-example.yml

```diff
@@ -26,7 +25,7 @@ services:
       POSTGRES_PASSWORD: postgres
 
   subquery-node:
-    image: acala/evm-subql:d9ee626
+    image: acala/evm-subql:0a99a2f
```

Comment:
need update again after merge

Reply:
```suggestion
    image: acala/evm-subql:0a99a2f
     image: acala/evm-subql:v2.5.22
```

Reply:
v2.5.22 will be latest, I am building now

Reply:
The `chainId` in `evm-subql/project.yaml` has been changed and needs to be merged and updated again